typespec-extension/package.json

```diff
@@ -51,8 +51,7 @@
     "@typespec/openapi": ">=0.51.0 <1.0.0",
     "@typespec/versioning": ">=0.51.0 <1.0.0",
     "@azure-tools/typespec-azure-core": ">=0.37.2 <1.0.0",
-    "@azure-tools/typespec-client-generator-core": ">=0.37.0 <1.0.0",
-    "@azure-tools/typespec-azure-resource-manager": ">=0.37.1 <1.0.0"
+    "@azure-tools/typespec-client-generator-core": ">=0.37.0 <1.0.0"
```

Comment:
Removed dependency on `typespec-azure-resource-manager`. It has dependency on `typespec-autorest` that we don't want.